aws/resource-count-aws.sh

```diff
@@ -68,6 +68,39 @@ else
    WITH_DATA="false"
 fi
 
+
+##########################################################################################
+## Optionally pass an AWS profile to use. Profiles are available at ~/.aws/config (Linux & Mac) or %USERPROFILE%\.aws\config (Windows)
```

Comment:
Rather than being optional, this forces the user to select a profile if more than profile one exists.

Reply:
Replied on the main thread, but this breaks it if run within cloudshell